ctapipe/reco/sklearn.py

```diff
@@ -222,6 +222,13 @@ def fit(self, key, table):
         y = self._table_to_y(table, mask=valid)
         self._models[key].fit(X, y)
 
+    def set_n_jobs(self, n_jobs):
+        """
+        Update n_jobs of all associated models.
+        """
+        for model in self._models.values():
+            setattr(model, "n_jobs", n_jobs)
```

Comment:
nonsensical use of setattr... why not use `mode.n_jobs = n_jobs`?

Reply:
True, this is not really necessary.
I started out with a general `set_model_attrs` function, but really we don't want to change anything else (at least I can't think of anything right now)